modules/claims_api/spec/workers/poa_form_builder_job_spec.rb

```diff
@@ -74,5 +74,63 @@
       expect_any_instance_of(ClaimsApi::PowerOfAttorneyPdfConstructor).to receive(:fill_pdf).twice.and_call_original
       subject.new.perform(power_of_attorney.id)
     end
+
+    it 'generates the expected pdf' do
+      Timecop.freeze(Time.zone.parse('2020-01-01T08:00:00Z'))
+      subject.new.perform(power_of_attorney.id)
+
+      path = Rails.root.join('tmp', "#{power_of_attorney.id}_final.pdf")
+      expected_path = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', '2122', 'expected_2122.pdf')
+
+      generated_md5s = pdf_to_md5s(path, power_of_attorney.id)
+      expected_md5s = pdf_to_md5s(expected_path, power_of_attorney.id)
+
+      expect(generated_md5s).to eq(expected_md5s)
+      File.delete(path) if File.exist?(path)
+      Timecop.return
+    end
+  end
+
+  private
+
+  # Converts PDF at path to array of md5's, one md5 for each page of the document.
+  # Problem: Taking an md5 of the PDF itself causes a mis-match between the generated and expected PDFs due to
+  # differences in metadata.
+  #
+  # @param [String] path The path to the PDF
+  # @param [String] identifier The unique identifier of the power of attorney
+  # @return [[String]] Array of md5's
+  def pdf_to_md5s(path, identifier)
```

Comment:
Can we consolidate this into some sort of spec helper for others to use?

Reply:
Updated!